src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -192,32 +192,32 @@ internal VectorToImageConvertingTransformer(IHostEnvironment env, params VectorT
 
         /// <param name="env">The host environment.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="height">The height of the output images.</param>
-        /// <param name="width">The width of the output images.</param>
+        /// <param name="imageHeight">The height of the output images.</param>
+        /// <param name="imageWidth">The width of the output images.</param>
         /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="colors">What colors to extract.</param>
-        /// <param name="order">In which order extracted colors presented in array.</param>
-        /// <param name="interleave">Whether the pixels are interleaved, meaning whether they are in <paramref name="order"/> order, or separated in the planar form, where the colors are specified one by one
+        /// <param name="colorsPresent">Specifies which <see cref="ImagePixelExtractingEstimator.ColorBits"/> are in present the input pixel vectors. The order of colors is specified in <paramref name="orderOfColors"/>.</param>
+        /// <param name="orderOfColors">The order in which colors are presented in the input vector.</param>
+        /// <param name="interleavedColors">Whether the pixels are interleaved, meaning whether they are in <paramref name="orderOfColors"/> order, or separated in the planar form, where the colors are specified one by one
         /// for all the pixels of the image. </param>
-        /// <param name="scale">Scale color pixel value by this amount.</param>
-        /// <param name="offset">Offset color pixel value by this amount.</param>
-        /// <param name="defaultAlpha">Default value for alpha color, would be overriden if <paramref name="colors"/> contains <see cref="ImagePixelExtractingEstimator.ColorBits.Alpha"/>.</param>
-        /// <param name="defaultRed">Default value for red color, would be overriden if <paramref name="colors"/> contains <see cref="ImagePixelExtractingEstimator.ColorBits.Red"/>.</param>
-        /// <param name="defaultGreen">Default value for grenn color, would be overriden if <paramref name="colors"/> contains <see cref="ImagePixelExtractingEstimator.ColorBits.Green"/>.</param>
-        /// <param name="defaultBlue">Default value for blue color, would be overriden if <paramref name="colors"/> contains <see cref="ImagePixelExtractingEstimator.ColorBits.Blue"/>.</param>
+        /// <param name="scalePixelColor">Scale color pixel value by this amount.</param>
+        /// <param name="offsetPixelCOlor">Offset color pixel value by this amount.</param>
```

Comment:
left a note below on whether those are correct. Offseting and scaling pixel color makes little sense to me, they make more sense in the image context.